.eslintrc.json

```diff
@@ -71,7 +71,6 @@
     "no-param-reassign": "off",
     "no-proto": "error",
     "no-restricted-properties": "error",
-    "no-return-assign": "error",
```

Comment:
I wouldn't remove this. Why can't we assign the value before the return call?

Reply:
I don't remember when I changed that, but feel free to change that 😉